TechTalk.JiraRestClient/ChangeLog.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
```

Comment:
Might worth the effort to clean up this file, like
* drop unnecessary namespace usings, and
* create separate files for al classes